blogs/tech-posts/_posts/2024-09-20-beyond-LLMs-rise-of-agentic-frameworks.md

```diff
@@ -0,0 +1,139 @@
+# Beyond LLMs: The Rise of Agentic Paradigms in AI
```

Comment:
Swaroop, this has to follow the MD blog format - Take the grafana blog as a reference.